src/postgres/getClientConfig.ts

```diff
@@ -12,13 +12,15 @@ import { invalidCredentialsErrorType } from "./tree/PostgresDatabaseTreeItem";
 import { PostgresServerTreeItem } from "./tree/PostgresServerTreeItem";
 
 export async function getClientConfig(treeItem: PostgresServerTreeItem, databaseName: string): Promise<ClientConfig> {
-    let username: string | undefined = treeItem.connectionString.username;
-    let password: string | undefined = treeItem.connectionString.password;
+    let username: string = treeItem.connectionString.username;
+    let password: string = treeItem.connectionString.password;
```

Comment:
Please revert all these username/password changes - I assume it's something that got accidentally pulled in from https://github.com/microsoft/vscode-cosmosdb/pull/1663?